README.md

```diff
@@ -8,9 +8,10 @@ Repo for the funding service design notification.
 Built with Flask.
 
 ## Prerequisites
+
 - python ^= 3.10
 
-# Getting started
+## Getting started
```

Comment:
I think this is wrong, getting started is a heading that has sub headings of installation, how to use etc

Pipelines should be at the same level I think. Happy to discus rearranging that, but it looked more correct previously to me

Reply:
thank you